front/src/components/Menu/index.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import IconButton from "@material-ui/core/IconButton";
+import Badge from "@material-ui/core/Badge";
+import ShoppingBasketIcon from "@material-ui/icons/ShoppingBasket";
+import StoreIcon from "@material-ui/icons/Store";
+import { Link } from "react-router-dom";
+
+import { selectBooks } from "features/books/booksSlice";
+
+import { useStyles } from "./styles";
+
+const Menu = ({ title }) => {
+  const books = useSelector(selectBooks);
+  const classes = useStyles();
+
+  return (
+    <React.Fragment>
+      <CssBaseline />
```

Comment:
to pasuje do App.js, jak najblizej roota, nie w components/Menu. Wszelkie configi mające wpływ globalnie dobrze trzymac tam, zamiast rozsiewac po apce

Reply:
Racja. 
Done.